testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -832,6 +832,28 @@ public void testDeleteAllRowsInPartition()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    public void testDeleteAllRowsUnpartitioned()
```

Comment:
i think the important aspect is "delete after delete".

assuming it's correct, let's reflect this in a test name: `testDeleteAfterDelete`

Reply:
Good idea; renamed.